module/documents/active-effect.mjs

```diff
@@ -123,6 +126,11 @@ export default class ActiveEffect5e extends ActiveEffect {
       return changes;
     }
 
+    else if ( (targetType === "string") && (change.mode === modes.OVERRIDE) && change.value.includes("{{}}") ) {
```

Comment:
Maybe just `{}` à la python? It's a bit closer to the `{foo}` format used in localisation strings that we might be able to expand later.